pubspec.yaml

```diff
@@ -1,14 +1,14 @@
 name: pana
 description: Evaluate the health and quality of a Dart package
-version: 0.12.13
+version: 0.12.13+1
 homepage: https://github.com/dart-lang/pana
 author: Dart Team <misc@dartlang.org>
 
 environment:
   sdk: '>=2.0.0 <3.0.0'
 
 dependencies:
-  analyzer: ^0.34.2
+  analyzer: '>=0.34.2 <0.36.0'
```

Comment:
^0.35.0

Reply:
was hoping to support a range of versions – or is that not how ya'll roll?

Reply:
There is no crime in supporting a range of versions..

But since `pana` is usually installed as a tool and not a dependency, what value does it bring.
Does it in any way out weight the cost of maintenance? Clearly, we're not checking the lower bounds :)

To be fair, I don't care, it's very unlikely anyone will install pana with anything but the latest version, since it's a usually installed as a tool with `pub global activate pana`.